precompile/contracts/rewardmanager/rewardmanagertest/bindings/RewardManagerTest.sol

```diff
@@ -0,0 +1,35 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import "./IRewardManager.sol";
```

Comment:
For some reason GitHub will not show this as a file move -- this was moved and renamed from `contracts/contracts/ExampleRewardManager.sol`, as a large amount of the remaining code in contracts has to be deleted.